2017/11.md

```diff
@@ -34,10 +34,18 @@ TBD Hotels nearby (optional)
 
 ## Agenda Topic Rules
 
-1. Proposals looking to advance must be added to the agenda along with necessary review materials 7 days prior to the first day of the meeting.
-   1. Note: any items added after 7 days prior to the start time of the meeting must be denoted with an ️:hourglass: symbol, which indicates that it is not eligible for stage advancement, and that it must wait until all other items have been addressed.
+Deadline for advancement eligibility: [**November 18th, 2017, 18:00 UTC**](https://www.timeanddate.com/countdown/generic?p0=1440&iso=20171118T18&msg=TC39%20Submission%20deadline)
+  - <sub>Note: this time is selected to be precisely 10 days prior to the start of the meeting</sub>
```

Comment:
The main goal is to have it defined with a strict precision. I don't want to block this anymore but I will write a follow up for mention it should also be a UTC -12 definition.

Reply:
My intention was for it to remain blocked until it got as close to 100% of delegate approvals as possible.

Reply:
More time for reviews is always helpful, and later means easier to contribute. This is a reasonable arbitrary point in the middle. Not sure why there's more need for rationale text here than elsewhere in the agenda.